### PR TITLE
Added Russian translation

### PR DESCRIPTION
**Pull request by KOLANICH**

### CODE REVIEW

File: l10n/gitahead_ru.ts [hunk 1]
```diff
@@ -0,0 +1,4999 @@
...
+    </message>
+    <message>
+        <location filename="../src/ui/AdvancedSearchWidget.cpp" line="90"/>
+        <source>Author name</source>
+        <translation>Имя автора</translation>
+    </message>
+    <message>
+        <location filename="../src/ui/AdvancedSearchWidget.cpp" line="91"/>
+        <source>Email:</source>
+        <translation>Электропочта:</translation>
```
Comment (ArtifeksNN) [inline l10n/gitahead_ru.ts:165]: Мне кажется лучше оставить просто email

File: l10n/gitahead_ru.ts [hunk 2]
```diff
@@ -0,0 +1,4999 @@
...
+    </message>
+    <message>
+        <location filename="../src/dialogs/AboutDialog.cpp" line="98"/>
+        <source>Privacy</source>
+        <translation>Шпионаж</translation>
+    </message>
+    <message>
+        <location filename="../src/dialogs/AboutDialog.cpp" line="109"/>
+        <source>Usage reporting has been disabled. Restart the application for changes to take effect.</source>
+        <translation>Шпионаж якобы был отключён. Перезапустите приложение для вступления изменений в силу.</translation>
```
Comment (ArtifeksNN) [inline l10n/gitahead_ru.ts:34]: Это шутка? Is it a joke?

File: l10n/gitahead_ru.ts [hunk 3]
```diff
@@ -0,0 +1,4999 @@
...
+    <message>
+        <location filename="../src/update/Updater_mac.mm" line="108"/>
+        <source>The new bundle could not be copied into place</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/update/Updater_mac.mm" line="117"/>
+        <location filename="../src/update/Updater.cpp" line="274"/>
+        <source>Helper application failed to start</source>
+        <translation>Помошник установщика не смог запуститься</translation>
```
Comment (ArtifeksNN) [inline l10n/gitahead_ru.ts:4863]: ПомоЩник

File: l10n/gitahead_ru.ts [hunk 4]
```diff
@@ -0,0 +1,4999 @@
...
+    </message>
+    <message>
+        <location filename="../src/update/Updater_mac.mm" line="96"/>
+        <source>The existing bundle could not be moved to the trash</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/update/Updater_mac.mm" line="108"/>
+        <source>The new bundle could not be copied into place</source>
+        <translation type="unfinished"></translation>
```
Comment (ArtifeksNN) [inline l10n/gitahead_ru.ts:4857]: ?

File: l10n/gitahead_ru.ts [hunk 5]
```diff
@@ -0,0 +1,4999 @@
...
+    </message>
+    <message>
+        <location filename="../src/dialogs/SettingsDialog.cpp" line="524"/>
+        <source>Tabs:</source>
+        <translation>Вкладки:</translation>
+    </message>
+    <message>
+        <location filename="../src/dialogs/SettingsDialog.cpp" line="526"/>
+        <source>Prompt:</source>
+        <translation type="unfinished"></translation>
```
Comment (ArtifeksNN) [inline l10n/gitahead_ru.ts:4996]: ?

File: l10n/gitahead_ru.ts [hunk 6]
```diff
@@ -0,0 +1,4999 @@
...
+    <message>
+        <location filename="../src/ui/ToolBar.cpp" line="884"/>
+        <location filename="../src/ui/ToolBar.cpp" line="1009"/>
+        <source>Show Log</source>
+        <translation>Показать журнал</translation>
+    </message>
+    <message>
+        <location filename="../src/ui/ToolBar.cpp" line="897"/>
+        <source>Diff View</source>
+        <translation>Смотреть diffы</translation>
```
Comment (ArtifeksNN) [inline l10n/gitahead_ru.ts:4691]: лучше просто diff

**Review by ArtifeksNN — COMMENTED** [review]
Очень фанатский перевод, много неточностей. Fan-made translation, many inaccuracies.

File: l10n/gitahead_ru.ts [hunk 7]
```diff
@@ -0,0 +1,4999 @@
...
+    </message>
+    <message>
+        <location filename="../src/dialogs/AboutDialog.cpp" line="76"/>
+        <source>Understand your history!</source>
+        <translation>Понимай свою историю!</translation>
+    </message>
+    <message>
+        <location filename="../src/dialogs/AboutDialog.cpp" line="96"/>
+        <source>Changelog</source>
+        <translation>Changelog</translation>
```
[the remark this answers is not in the document]
Comment (KOLANICH) [reply l10n/gitahead_ru.ts:19]: Maybe.

File: l10n/gitahead_ru.ts [hunk 8]
```diff
@@ -0,0 +1,4999 @@
...
+    </message>
+    <message>
+        <location filename="../src/dialogs/AboutDialog.cpp" line="97"/>
+        <source>Acknowledgments</source>
+        <translation>Посвящения и спасибки</translation>
```
[the remark this answers is not in the document]
Comment (KOLANICH) [reply l10n/gitahead_ru.ts:24]: Thanks, this may be the closest analogue.

File: l10n/gitahead_ru.ts (hunk 1 above)
Comment (KOLANICH) [reply l10n/gitahead_ru.ts:165]: TortoiseGit uses this (the translation is also by me :). `Э-почта` looks weird, `электронная` is too long.

File: l10n/gitahead_ru.ts (hunk 2 above)
Comment (KOLANICH) [reply l10n/gitahead_ru.ts:34]: A  Russian proverb says that "every joke has [only] a part of joke."

File: l10n/gitahead_ru.ts (hunk 3 above)
Comment (KOLANICH) [reply l10n/gitahead_ru.ts:4863]: Thanks

File: l10n/gitahead_ru.ts (hunk 4 above)
Comment (KOLANICH) [reply l10n/gitahead_ru.ts:4857]: The translation is unfinished. I have translated only short texts.

File: l10n/gitahead_ru.ts (hunk 6 above)
Comment (KOLANICH) [reply l10n/gitahead_ru.ts:4691]: Maybe, depends on context.

File: l10n/gitahead_ru.ts (hunk 2 above)
Comment (KOLANICH) [reply l10n/gitahead_ru.ts]: I guess just Почта is OK.

File: l10n/gitahead_ru.ts (hunk 4 above)
Comment (KOLANICH) [reply l10n/gitahead_ru.ts]: Fixed.

File: l10n/gitahead_ru.ts (hunk 5 above)
Comment (KOLANICH) [reply l10n/gitahead_ru.ts:5006]: IDK how to translate thi in this context. So let it be untranslated for now.

File: l10n/gitahead_ru.ts (hunk 6 above)
Comment (KOLANICH) [reply l10n/gitahead_ru.ts]: Done.

**Pull request merged**